pages/end.html

```diff
@@ -18,9 +18,12 @@
     <div class="container">
       <div id="end" class="flex-center flex-column">
         <h1 id="finalScore">0</h1>
+        <div style="color: white; font-size: 30px; margin-bottom: 1rem;"><p>You got <span id="noScore" style="font-size: 30px;">0</span>/15 correct.</p></div>
```

Comment:
Some quizzes only have 5, 7, 10, questions and not always 15, so we need to define it per quiz. @sai140202